frontend/utils.js

```diff
@@ -29,9 +29,9 @@ function removeEndingSlash(val) {
   return val
 }
 
-export const API_BASE_URL =
-  removeEndingSlash(getConfig().publicRuntimeConfig.API_BASE_URL) ||
-  'https://pennclubs.com/api'
+export const DEFAULT_SITE_ORIGIN = process.env.NODE_ENV === 'production' ? 'https://pennclubs.com' : `http://localhost:${process.env.PORT || 3000}`
```

Comment:
Can we use the `DOMAIN` env variable here? It will be set to a domain without the http(s) ex. `pennclubs.com`